### PR TITLE
Improve cli help text

### PR DESCRIPTION
**Pull request by amacneil**
**Public-Facing Changes**
The CLI help text was very bland, I improved it.

### CODE REVIEW

File: go/cli/mcap/cmd/root.go [hunk 1]
```diff
@@ -12,7 +12,7 @@ var cfgFile string
 
 var rootCmd = &cobra.Command{
 	Use:   "mcap",
-	Short: "A small utility for demonstration of the mcap file format",
+	Short: "\U0001F52A Officially the top-rated CLI for managing MCAP files.",
```
Comment (jtbandes) [inline go/cli/mcap/cmd/root.go:15]:
```suggestion
	Short: "\U0001F54C A CLI tool for managing MCAP files.",
```

Comment (amacneil) [reply go/cli/mcap/cmd/root.go:15]: From PR description

> The CLI help text was very bland, I improved it

Comment (amacneil) [reply go/cli/mcap/cmd/root.go:15]: Alternatively:

> 🔪 A chef's knife for slicing and dicing MCAP files

Comment (jtbandes) [reply go/cli/mcap/cmd/root.go:15]: 🤷

Could also randomly select one of several phrases on each invocation.

Comment (amacneil) [reply go/cli/mcap/cmd/root.go:15]: Heh - I thought about that but it looks like it only accepts a string, I'm not sure how to customize it at runtime.

**Pull request merged**